src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraSecret.kt

```diff
@@ -7,7 +7,8 @@ data class AuroraSecretVault @JvmOverloads constructor(
         val name: String,
         val secrets: Map<String, String>,
         val permissions: AuroraPermissions? = null,
-        val versions: Map<String, String?> = mapOf()
+        val versions: Map<String, String?> = mapOf(),
+        val canEdit: Boolean = false
```

Comment:
Jeg er ikke så veldig begeistret for denne løsningen. Vault-objektene blir nå "personlige" og dette er en viral egenskap som gjør at f.eks. DeploymentBundle også blir et personlig objekt. Etter min mening bør man prøve holde rettighetsstyring "ortogonalt" til øvrig forretningslogikk.

Reply:
Vi kan endre dette senere. Jeg er enig. Men akkurat nå må det ut.